webapp/app/data_access/form_data_controller.py

```diff
@@ -0,0 +1,35 @@
+import redis
+import redislite
+from app.config import Config
+from pydantic import MissingError
+
+redis_url = Config.SESSION_DATA_STORAGE_URL
+if redis_url == 'None':
+    r = redislite.StrictRedis()
+else:
+    r = redis.Redis.from_url(Config.SESSION_DATA_STORAGE_URL)
```

Comment:
Personally, I would prefer to have a function that returns the redis connection because this does make it a) easier to exchange the redis in tests b) it makes it more explicit what happens and that the redis is something that can change 

```suggestion
def get_redis_connection():
    if redis_url == 'None':
        return redislite.StrictRedis()
    else:
        return redis.Redis.from_url(Config.SESSION_DATA_STORAGE_URL)
```

Reply:
I think we should also try to avoid single letter variable names as these become hard to understand if they are taken out of context.

Reply:
I agree! Thanks for the suggestion